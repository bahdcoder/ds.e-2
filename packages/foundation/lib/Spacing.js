const spaces = {
    none: 'none',
    xxxs: 'xxxs',
    // 4px
    xxs: 'xxs',
    // 8px
    xs: 'xs',
    // 12px
    sm: 'sm',
    // 16px
    md: 'md',
    // 24px
    lg: 'lg',
    // 32px
    xl: 'xl',
    // 48px
    xxl: 'xxl',
    // 72px
    xxxl: 'xxxl',
};
export default Object.freeze(spaces);
