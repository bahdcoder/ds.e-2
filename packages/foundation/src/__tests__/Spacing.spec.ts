import Spacing from '../Spacing'

test('should match snapshot', () => {
    expect(Spacing).toMatchSnapshot()
})
