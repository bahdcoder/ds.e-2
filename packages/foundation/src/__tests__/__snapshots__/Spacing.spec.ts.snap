// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "lg": "lg",
  "md": "md",
  "none": "none",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
